pinot-broker/src/main/java/com/linkedin/pinot/broker/broker/helix/HelixBrokerStarter.java

```diff
@@ -188,13 +188,17 @@ public void run() {
     return brokerServerBuilder;
   }
 
-  private String getZkAddressForBroker(String zkServers, String helixClusterName) {
+  public static String getZkAddressForBroker(String zkServers, String helixClusterName) {
     List tokens = new ArrayList<String>();
-
-    for (String token : zkServers.split(",")) {
-      tokens.add(StringUtil.join("/", StringUtils.chomp(token, "/"), helixClusterName, PROPERTY_STORE));
+    String[] zkSplit = zkServers.split("/", 2);
```

Comment:
Actually, I think this method is completely unnecessary. It's only used in `HelixBrokerStarter` to create another ZKClient that points only to the propertystore, instead of using the same connection as the `HelixManager` (since the helix manager is started later).

My guess is that it would be better to reuse the same ZK connection as the helix manager and get the property store from it.